docs/configuration/upstream/tests.md

```diff
@@ -511,6 +511,42 @@ execute:
 
 Packit Service communicates with Testing Farm via its [API][testing-farm-api].
 
+### Public IDs of Packit tokens
+
+:::caution
+
+This feature has not been tested and verified by Packit Team yet.
+
+:::
```

Comment:
can'we remove this now?

Reply:
I think we have different definitions of “verified”, but I guess I could remove it :D